spec/unit/recipes/default_spec.rb

```diff
@@ -32,4 +27,47 @@
         block_size\s512.*/x)
     end
   end
+
+  context 'On Windows 2012R2' do
+    let(:chef_run) do
+      runner = ChefSpec::SoloRunner.new(platform: 'windows',
+                                        version: '2012R2',
+                                        file_cache_path: 'C:\chef\cache') do |node|
+        node.set['snapraid']['content_files'] = ['D:\\']
+        node.set['snapraid']['parity_disks'] = ['D:\\']
+        node.set['snapraid']['data_disks'] = ['D:\\']
+        node.set['snapraid']['block_size'] = '512'
+        node.set['snapraid']['install_directory'] = 'D:\SnapRaid'
+        node.set['snapraid']['config_directory'] = 'D:\SnapRaid\Config'
+        node.set['snapraid']['url'] = 'https://github.com/amadvance/snapraid/releases/download/v9.1/snapraid-9.1-windows-x64.zip'
+      end
+
+      runner.converge(described_recipe)
+    end
+
+    it 'downloads snapraid' do
+      expect(chef_run).to create_remote_file('C:\chef\cache\snapraid-9.1-windows-x64.zip')
+      resource = chef_run.remote_file('C:\chef\cache\snapraid-9.1-windows-x64.zip')
+      expect(resource).to notify('windows_zipfile[D:\SnapRaid]').to(:unzip).immediately
+    end
+
+    it 'defines unzip of snapraid with action nothing' do
+      expect(chef_run).to_not unzip_windows_zipfile_to('D:\SnapRaid')
+    end
+
+    it 'creates directories' do
+      expect(chef_run).to create_directory('D:\SnapRaid')
+      expect(chef_run).to create_directory('D:\SnapRaid\Config')
+    end
+
+    it 'creates the correct configuration file' do
+      expect(chef_run).to create_template('D:\SnapRaid\Config/snapraid.conf')
+      expect(chef_run).to render_file('D:\SnapRaid\Config/snapraid.conf').with_content(/
```

Comment:
Huh, in playing about with this, I've found that multiple appended .with_content()s don't seem to actually evaluate, and as a result the test always returns success!
I've replaced with 
```
expect(chef_run).to render_file('/etc/snapraid.conf').with_content { |content|
        expect(content).to match(%r{parity /mnt/virtual/snapraid.parity})
        expect(content).to match(%r{content /mnt/virtual/snapraid.content})
        expect(content).to match(%r{disk d1 /mnt/virtual})
        expect(content).to match(/block_size 512/)
      }
```